### ISSUE TITLE
Add Website to top banner of githup repository

### ISSUE THREAD

**Issue by JLLeitschuh** [TestFX/TestFX#232]
This would just make accessing resources for this project much faster.
Thanks!

**Comment by hastebrot** [comment]
Yes, good idea. But the website isn't ready yet. Any help is welcome.

**Comment by JLLeitschuh** [comment]
Haha, the good ol' PR's please message. I know that all too well from the projects I manage. If I get the time I will but I've got a lot on my plate right now.
Nice job with this project anyways.
